lab-cameron/src/components/search-form.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+class SearchForm extends React.Component {
```

Comment:
this component is where you should be handling your Reddit API call, not inside of App. App should just be holding the state and rendering the other components